test/dlc_tests/ecs/pytorch/training/test_ecs_pytorch_training.py

```diff
@@ -89,7 +89,7 @@ def test_ecs_pytorch_training_mnist_gpu(
 @pytest.mark.parametrize("ecs_instance_type", ["c5.9xlarge"], indirect=True)
 @pytest.mark.parametrize("ecs_ami", [ECS_AML2_CPU_USWEST2], indirect=True)
 def test_ecs_pytorch_s3_plugin_training_cpu(
-    cpu_only, ecs_container_instance, pytorch_training, training_cmd, ecs_cluster_name, pt17_and_above_only
+    ecs_container_instance, pytorch_training, training_cmd, ecs_cluster_name, outside_versions_skip
```

Comment:
Lets add the `cpu_only` fixture back

Reply:
is `outside_versions_skip` declared in the test function definition work? i think it is redundant